### PR TITLE
dfmt 0.5.0-beta3 (devel)

### PR DESCRIPTION
**Pull request by John-Colvin**

### CODE REVIEW

File: Library/Formula/dfmt.rb [hunk 1]
```diff
@@ -4,6 +4,10 @@ class Dfmt < Formula
   url "https://github.com/Hackerpilot/dfmt.git",
       :tag => "v0.4.4",
       :revision => "c53eb5e0e4a6bb494dccc1eb1ff3e6f4dc820bfa"
+  
+  devel "https://github.com/Hackerpilot/dfmt.git",
```
Comment (DomT4) [inline Library/Formula/dfmt.rb]: Needs to be `devel do`, not just `devel`. Should be beneath the `bottle do` block as well :wink:

Comment (John-Colvin) [inline Library/Formula/dfmt.rb]: done

File: Library/Formula/dfmt.rb [hunk 2]
```diff
@@ -13,6 +13,12 @@ class Dfmt < Formula
     sha256 "d306278e22b46e60b6474593e1107db753c4fefc907e66bd1f348cc29af0fc95" => :mavericks
   end
 
+  devel do
+    url "https://github.com/Hackerpilot/dfmt.git",
+        :tag => "v0.5.0-beta3",
+        :revision => "845358bb61603031b0817aed03097064c8f2553f"
+  end
```
Comment (DomT4) [inline Library/Formula/dfmt.rb]: Apologies but you may have to set an explicit `version` here as well to include the `beta` tag. It's being filtered out at the moment:
```
dfmt: stable 0.4.4 (bottled), devel 0.5.0, HEAD
```

Comment (John-Colvin) [inline Library/Formula/dfmt.rb]: fixed

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/cff0c73f0c25023e9a4b621103ae638f60a216c6. Thank you for another contribution to Homebrew @John-Colvin! :smiley:

**Pull request closed without merging**